apps/omg_watcher/test/omg_watcher/integration/invalid_exit_test.exs

```diff
@@ -19,12 +19,14 @@ defmodule OMG.Watcher.Integration.InvalidExitTest do
   use OMG.ChildChain.Integration.Fixtures
   use Plug.Test
 
-  alias OMG.Utxo
-  require Utxo
   alias OMG.Eth
+  alias OMG.Utxo
   alias OMG.Watcher
-  alias OMG.Watcher.{Event, TestHelper}
   alias OMG.Watcher.Integration.TestHelper, as: IntegrationTest
+  alias OMG.Watcher.{Event, TestHelper}
```

Comment:
```suggestion
  alias OMG.Watcher.Event
  alias OMG.Watcher.TestHelper
```

also, since it's support, you might as well move it around and rename it properly